tasks/palindrome/CheckPalindrome.java

```diff
@@ -0,0 +1,28 @@
+package tasks.palindrome;
+
+import java.io.Console;
+
+public class CheckPalindrome {
+    public static void main(String[] args) {
+        Console console = System.console();
+        String response = "";
+
+        do {
+            char[] word = console.readLine("Please enter a string to check if it's palindrome.\n").toCharArray();
+            int index = 0;
+            boolean isPalindrome = false;
+
+            for (int i = word.length - 1; i >= (word.length / 2); i--) {
+                isPalindrome = word[i] == word[index];
```

Comment:
Awesome work, my only suggestion is that we don't need keep processing the string if we found a that is not palindrome, to fix it you can add something like this:

```java
isPalindrome = word[i] == word[index];
if (!isPalindrome) {
   break;
}
```